packages/tinacms/src/plugins/fields/NumberFieldPlugin.tsx

```diff
@@ -16,14 +16,18 @@ limitations under the License.
 
 */
 
+import * as React from 'react'
 import { wrapFieldsWithMeta } from './wrapFieldWithMeta'
-import { NumberInput } from '@tinacms/fields'
-import { parse } from './numberFormat';
+import { NumberInput as BaseNumberInput, NumberProps } from '@tinacms/fields'
+import { parse } from './numberFormat'
 
-export const NumberField = wrapFieldsWithMeta(NumberInput)
+export const NumberField = wrapFieldsWithMeta<
+  { placeholder: string },
+  NumberProps
+>(() => <BaseNumberInput />)
```

Comment:
Nothing is being passed to the BaseNumberInput. That's what's causing the errors.
```suggestion
>(BaseNumberInput)
```